### PR TITLE
🎉 Migrate Hellobaton Source Connector to Low Code

### LINKED ISSUE #29139: hellobaton

### PR DESCRIPTION
**Pull request by topefolorunso**
## What
Migrating Source Hellobaton to Low-Code resolves #29139

## How
Created the connector using the connector buider UI, then imported the manifest.yaml to the low code cdk.

## Recommended reading order
1. `manifest.yaml`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: docs/integrations/sources/baton.md
Comment (sajarin) [inline docs/integrations/sources/baton.md:1]: Can you rename this file to match the name of the connector?

File: airbyte-integrations/connectors/source-hellobaton/setup.py [hunk 1]
```diff
@@ -11,8 +11,9 @@
 
 TEST_REQUIREMENTS = [
     "requests-mock~=1.9.3",
-    "pytest~=6.1",
+    "pytest~=6.2",
     "pytest-mock~=3.6.1",
+    "connector-acceptance-test",
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-hellobaton/setup.py:16]: don't think connector-acceptance-test is needed here

File: airbyte-integrations/connectors/source-hellobaton/source_hellobaton/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  - "#/definitions/project_attachments_stream"
+  - "#/definitions/projects_stream"
+  - "#/definitions/tasks_stream"
+  - "#/definitions/task_attachments_stream"
+  - "#/definitions/templates_stream"
+  - "#/definitions/time_entries_stream"
+  - "#/definitions/users_stream"
+
+spec:
+  documentation_url: https://docs.airbyte.com/integrations/sources/baton
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-hellobaton/source_hellobaton/manifest.yaml:150]: can you update this doc url?

File: airbyte-integrations/connectors/source-hellobaton/source_hellobaton/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  type: Spec
+  connection_specification:
+    $schema: http://json-schema.org/draft-07/schema#
+    type: object
+    additionalProperties: false
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-hellobaton/source_hellobaton/manifest.yaml:155]: can you set this to true and skip backward incompatibility check in the acceptance-test-config.yml for the spec task?

**Review by sajarin — CHANGES_REQUESTED** [review]
Thanks for the PR, left some comments

**Comment by topefolorunso** [comment]
Changes made @sajarin, please review.

File: airbyte-integrations/connectors/source-hellobaton/acceptance-test-config.yml
Comment (sajarin) [inline airbyte-integrations/connectors/source-hellobaton/acceptance-test-config.yml:1]: Can you migrate this file to use the latest formatting for an acceptance-test-config.yml. Other connectors may be a good reference

**Review by sajarin — CHANGES_REQUESTED** [review]
Just a few more things: 

1. Can you update all of the schema files and make each of the properties nullable? 
2. See below

File: airbyte-integrations/connectors/source-hellobaton/acceptance-test-config.yml
Comment (topefolorunso) [reply airbyte-integrations/connectors/source-hellobaton/acceptance-test-config.yml:1]: Done @sajarin

**Comment by topefolorunso** [comment]
> Just a few more things:
> 
> 1. Can you update all of the schema files and make each of the properties nullable?
> 2. See below

Changes made @sajarin, please review

**Review by sajarin — APPROVED** [review]

**Pull request merged**